EU4ToVic2Tests/MapperTests/CultureGroups/CultureTests.cpp

```diff
@@ -0,0 +1,172 @@
+#include "CultureGroups/Culture.h"
+#include "gtest/gtest.h"
+
```

Comment:
Add tests (or a test) that demonstrate(s) the default values for the different variables.